docs/community/working-group-definitions.md

```diff
@@ -0,0 +1,195 @@
+it's likely that the Language WG will work especially close with them,
+particularly given the long-lasting effects of language decisions.
+
+### Members
+
+* @JamesWTruher
+* @rjmholt
+* @daxian-dbw
+* @BrucePay
+
+## Remoting
+
+The Remoting WG should focus on topics like the **PowerShell Remoting Protocol (PSRP)**,
+the **protocols implemented under PSRP** (e.g. WinRM and SSH),
+and **other protocols used for remoting** (e.g. "pure SSH" as opposed to SSH over PSRP).
+Given the commonality of serialization boundaries, the Remoting WG should also focus on
+**the PowerShell job system**.
+
+### Members
+
+* @PaulHigin
+* @SteveL-MSFT
+* @TravisEz13
+* @anmenaga
+
+## Modules
+
+For those core/inbox modules whose source code lives within the `PowerShell/PowerShell` repository,
+we should have some set of WGs responsible for their cmdlet design guidelines and enforcement,
+as well as to manage the proposal of new cmdlets or breaking changes.
+
+However, some modules that ship as part of the PowerShell package are managed in other source repositories.
+These modules should be owned by the maintainers of those individual repositories.
+These modules include:
+
+* [`Microsoft.PowerShell.Archive`](https://github.com/PowerShell/Microsoft.PowerShell.Archive)
+* [`PackageManagement` (formerly `OneGet`)](https://github.com/OneGet/oneget)
+* [`PowerShellGet`](https://github.com/PowerShell/PowerShellGet)
+* [`PSDesiredStateConfiguration`](https://github.com/PowerShell/xPSDesiredStateConfiguration)
+  (Note: this community repository maintains a slightly different version of this module on the Gallery,
+  but should be used for future development of `PSDesiredStateConfiguration`.)
+* [`PSReadLine`](https://github.com/PowerShell/PSReadLine)
+* [`ThreadJob`](https://github.com/PowerShell/Modules/tree/master/Modules/Microsoft.PowerShell.ThreadJob)
+
+In the long-term, our desire is to manage all modules using the "separate repo" approach,
+but we recognize that we may need WGs for many "inbox" modules,
+and that some modules may never be able to be abstracted out of `PowerShell/PowerShell`
+(e.g. `Microsoft.PowerShell.Core`).
+However, until it is determined how to handle these distribute these modules across WGs
+(or whether they should be moved out to other repos), the following modules will be owned
+by the Maintainers:
+
+* CimCmdlets
+* Microsoft.PowerShell.Diagnostics
+* Microsoft.PowerShell.Host
+* Microsoft.PowerShell.Management
+* Microsoft.PowerShell.Security
+* Microsoft.PowerShell.Utility
+* Microsoft.WSMan.Management
+* PSDiagnostics
+* Web/JSON Cmdlets
+
+## Security
+
+The Security WG should be brought into any issues or pull requests which may have security implications
+in order to provide their expertise, concerns, and guidance.
+
+### Members
+
+* @TravisEz13
+* @PaulHigin
+
+## Explicitly not Working Groups
+
+Some areas of ownership in PowerShell specifically do not have Working Groups.
+For the sake of completeness, these are listed below:
```

Comment:
This sounds like "we don't accept PRs in the areas" but it is not so - we need to say welcome explicitly.

Reply:
```suggestion
## Explicitly not Working Groups

Some areas of ownership in PowerShell specifically do not have Working Groups
We're still happy to take issues and pull requests in these areas,
which should generally follow the same process as feature code
(minus the RFC process and Committee involvement).
For the sake of completeness, these are listed below:
```